lib/ui/lifecycle_event_handler.dart

```diff
@@ -19,7 +19,7 @@ class LifecycleEventHandler extends WidgetsBindingObserver {
         }
         break;
       case AppLifecycleState.inactive:
-      case AppLifecycleState.hidden:
+      // case AppLifecycleState.hidden:
```

Comment:
Why is this commented out? Check out to the latest version of Flutter if it's breaking compilation.

Reply:
Cool I updated my Flutter version, and uncommented it